roles/validate_inventory/tasks/network.yml

```diff
@@ -18,4 +18,4 @@
   shell: # noqa 305
     cmd: "ping -c 1 -W 2 {{ ntp_server }}"
   changed_when: False
-  when: (setup_ntp_service | default(True)) != True
+  when: setup_ntp_service | default(True)
```

Comment:
This change is inverting the (admittedly confusing) logic which I don't think is right, but on consideration I think this `when` should be entirely removed. The `{{ ntp_server }}` referenced must always be set so pinging it is always a valid check.
The `setup_ntp_service` flag only indicates if the host is already running the NTP service, not if the host is pingable.

Please also change the task name to match.